src/ui/styles/Overlay.module.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+.Overlay {
+  background-color: #282c34;
+  color: white;
+  font-size: calc(10px + 2vmin);
```

Comment:
This is going to scale up the font size based on the height of the screen. Is that what you want?